experimental/c-cpp/runner.py

```diff
@@ -251,6 +251,15 @@ def run_on_targets(target,
               build_heuristics=build_heuristics,
               generator_heuristics=generator_heuristics)
 
+  # Cleanup the OSS-Fuzz docker image
+  clean_up_cmd = [
+      'docker', 'image', 'rm', f'gcr.io/oss-fuzz/{worker_project_name}'
```

Comment:
hmm, I thought we were always running `docker run` with `--rm` which should do this cleanup automatically on exit. Is that not the case?

Reply:
We build an oss-fuzz project per project that is being analysed. It's the image build from the following that we need to clean up: https://github.com/google/oss-fuzz-gen/blob/4e3798ef7a6af994d7e09094cd399876e701d1a2/experimental/c-cpp/runner.py#L67-L72

i.e. after the execution there will be in the shared folders:
```
build/out/temp-project-0
build/out/temp-project-1
...
```
each corresponding to auto-generation of a given repository

Reply:
I thought `docker image rm` removes the image while `docker run --rm` removes the container?